docs/docs/policies.mdx

```diff
@@ -363,11 +382,15 @@ policies:
       - action: utter_greet  # `utter_greet` contradicts `utter_chitchat` from the rule above
     ```
  * `restrict_rules` (default: `true`): Rules are restricted to one user turn, but
-    there can be multiple bot events, including e.g. a form being filled and its
+    there can be multiple bot events, including e.g. a form being filled and its subsequent submission.
     Change this parameter to `false` at your own risk.
```

Comment:
Can we rephrase this 😅

Reply:
what about "Changing this parameter to `false` may result in unexpected behaviors."?

Reply:
Sounds much better 😄